### DISCUSSION TITLE
Handle and Organise Posts by Different Authors and Contributors

### DISCUSSION THREAD

**Discussion by sirinath** [11ty/eleventy#4050] [Enhancement Queue]
This relates to https://github.com/11ty/eleventy/issues/959 and https://github.com/11ty/eleventy/issues/955.

If multiple authors are contributing to a blog then:
- one should have a collection of posts and metadata about the author as a collection to generate
  - list of authors by
    - 1st letter of the name
    - topics written / areas covered - using taxonomies
  - author bio pages - no post templates / [sections](https://gohugo.io/content-management/sections/)
  - list of posts by authors

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
Cross-link to #4051
